web-ui/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 node {
-    version = '14.17.6' // https://nodejs.org/en/
+    version = '16.20.2' // https://nodejs.org/en/
```

Comment:
So I worked on this change with Kimberlin. The reason it is in this PR is because it was actually something that was breaking on my machine. It was causing a failure to build in gradle. I'm not opposed to cherry-picking this and merging it in on it's own though. I'll leave the decision up to @mkimberlin

Reply:
I'm not too concerned about this, tbh. Yes, ideally I would break it out...but I don't want to make work over it.

Reply:
Are we going to update `SETUP_MAC.md` with this?